src/cmd/script.rs

```diff
@@ -104,6 +104,18 @@ pub fn new<'a, 'b>() -> App<'a, 'b> {
                 .long("no-abort-on-error")
                 .help("Do not abort analysis/compilation on first caught error (only for programs that support early aborting)")
         )
+        .arg(
+            Arg::with_name("compilation_mode")
+                .long("compilation_mode")
```

Comment:
How about changing the long name to `compilation-mode` to stay in line with the other multi-word switches?

Reply:
Ok, I changed long name as you suggest